README.md

```diff
@@ -144,6 +144,7 @@ You can pass the following options to `cmake ..`:
 
 * `-DBUILD_SAMPLE` -- Build the sample executables. ON by default.
 * `-DIOT_CORE_ENABLE_CREDENTIALS` -- Build the sample applications using IoT credentials. OFF by default.
+* `-DENABLE_DATA_CHANNEL` -- Build SDK & Samples with data channel. ON by default.
```

Comment:
(nit) lowercase "samples" for consistency.

Reply:
Adjusted